documentation/snapshot/docs/install/cli.md

```diff
@@ -220,3 +220,12 @@ Options `--stdin` and `--patterns-from-stdin` are mutually exclusive, only one o
 
 !!! tip "Microsoft Windows"
     On Microsoft Windows you'll have to use `java -jar ktlint ...`.
+
+!!! tip "Microsoft Windows"
```

Comment:
I see this is double, will need to fix this. as well as removing the extra jar file

Reply:
Very recently, it is not yet released, we have also added a "ktlint.bat" file for windows to support Java 16+ usage  (#2059). Please update above to using the batch file. But also keep the "java -jar" for hard core CLI users that do not want to use a provided batch file.